src/app/beer_garden/db/api.py

```diff
@@ -8,8 +10,25 @@
 count = beer_garden.db.mongo.api.count
 query_unique = beer_garden.db.mongo.api.query_unique
 query = beer_garden.db.mongo.api.query
-create = beer_garden.db.mongo.api.create
-update = beer_garden.db.mongo.api.update
-delete = beer_garden.db.mongo.api.delete
 reload = beer_garden.db.mongo.api.reload
 replace_commands = beer_garden.db.mongo.api.replace_commands
+
+
+@publish_event(Events.DB_CREATE)
+def create_function(obj):
+    return beer_garden.db.mongo.api.create(obj)
+
+
+@publish_event(Events.DB_CREATE)
+def update_function(obj):
+    return beer_garden.db.mongo.api.update(obj)
+
+
+@publish_event(Events.DB_DELETE)
+def delete_function(obj):
+    return beer_garden.db.mongo.api.delete(obj)
```

Comment:
If we call these something like `_create` it'll be more apparent that they aren't meant to be accessed outside of this module.

Reply:
Agreed, I will include these changes